source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -46,54 +46,74 @@ Create a Device Sync-Enabled App
    .. step:: Enable Device Sync or Modify the Template Configuration
 
       If you have not started with a template app, :ref:`enable Device Sync 
-      <enable-sync>` with Flexible Sync. Enable :ref:`Anonymous authentication 
-      <anonymous-authentication>`. Configure :ref:`permissions 
+      <enable-sync>` with Flexible Sync. Configure :ref:`permissions 
       <device-sync-compatibility>` to allow anyone to read and write all data.
 
-      If you started with a template app, enable :ref:`Anonymous authentication 
-      <anonymous-authentication>`. Go to :guilabel:`Rules`, and delete the 
-      existing rule for the ``todo`` / ``Item`` collection. Replace it with 
+      The Tiered Sync server uses :ref:`Anonymous Authentication 
```

Comment:
The Tiered Sync server currently uses anonymous authentication to authenticate with the App Services App. In an earlier version, you had to enable Anonymous Authentication to use it, but my understanding is that now we're enabling anonymous authentication automatically somehow when you set up Tiered Sync. It was suggested to remove this entirely, but I thought a customer might find it weird to use an app where email/password auth was enabled - and if they happened to look at auth providers again after setting up Tiered Sync, unexpectedly see Anonymous Auth enabled. I was trying to proactively provide the explanation to avoid surprise. But if you think this reads confusing, I can remove it.